pages/admin/AdminFunctionsHandler.inc.php

```diff
@@ -109,6 +109,31 @@ function clearDataCache() {
 
 		Request::redirect(null, 'admin');
 	}
+
+	/**
+	 * Download scheduled task execution log file.
+	 * @param $args array
+	 */
+	function downloadScheduledTaskLogFile() {
+		$this->validate();
+		$application =& Application::getApplication();
+		$request =& $application->getRequest();
+
+		$file = $request->getUserVar('file');
```

Comment:
only if a developer changes the code. The variable file here is only the file name, not the complete path. The scheduled task helper will only get this filename and search for it inside the scheduled task folder, in private ojs files folder. So basically one can download everything that's inside the scheduled tasks logs folder, which is the purpose.